tuftool/Cargo.toml

```
@@ -21,19 +21,21 @@ aws-sdk-kms = "0.28"
 aws-sdk-ssm = "0.28"
 chrono = { version = "0.4", default-features = false, features = ["alloc", "std", "clock"] }
 clap = { version = "4", features = ["derive"] }
+futures = "0.3.28"
```

Can you reduce the specificity of the dependency versions? We have seen conflicts with other downstream dependencies at times when they require a "no-higher-than X" version.

```suggestion
futures = "0.3"
```